### DISCUSSION TITLE
Osu

### DISCUSSION THREAD

**Discussion by lrxyab** [catppuccin/catppuccin#1861] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes (but might take me months due to entrance exams)

**Additional context 📷 (optional)**
relevant forum [ https://osu.ppy.sh/community/forums/15]( https://osu.ppy.sh/community/forums/15)
[skin template](https://drive.google.com/file/d/1bIX2rYlJgrHoQysMPEr4GCt399MbnWFZ/view?usp=sharing)

**Comment by AnubisNekhet** [comment]
I don't think there's a good way to approach this since hitcircles are veeery subjective and we can't just recolor the base UI since it looks horrible. That leaves us with reskinning other skins or making our own, which takes a lot of effort and time.

**Comment by yavko** [comment]
> I don't think there's a good way to approach this since hitcircles are veeery subjective and we can't just recolor the base UI since it looks horrible. That leaves us with reskinning other skins or making our own, which takes a lot of effort and time.

You could try skinning the new default  lazer skin, argon

**Comment by AnubisNekhet** [comment]
> You could try skinning the new default lazer skin, argon
Argon is still pretty wip but I guess we could skin it once lazer is in beta

**Comment by yavko** [comment]
But lazer works just fine, has linux support and is open source. Plus afaik development has slowed down on stable, and i think for the future it will better just to do for lazer.

**Comment by 10kmotorola** [comment]
bump :3

**Comment by Nekosis** [comment]
https://www.reddit.com/r/OsuSkins/comments/zmddeh/catppuccin_hd_169_std_animated/
OP here might be interested in making this an official port

**Comment by shadow-absorber** [comment]
using the one linked above works but would like more flavours of catppuccin for osu laser
